snuba/web/rpc/v1/resolvers/R_eap_spans/resolver_get_trace.py

```diff
@@ -168,14 +195,14 @@ def _value_to_attribute(key: str, value: Any) -> tuple[AttributeKey, AttributeVa
                 val_double=value,
             ),
         )
-    elif isinstance(value, str):
+    elif isinstance(value, str) or isinstance(value, datetime):
         return (
             AttributeKey(
                 name=key,
                 type=AttributeKey.Type.TYPE_STRING,
             ),
             AttributeValue(
-                val_str=value,
+                val_str=str(value),
```

Comment:
timestamps should be converted to the protobuf TimeStamp object, not utc strings

Reply:
I'd suggest to return timestamps as unix timestamp for now since we don't have a date time type in the RPC yet.